src/main/php/PDepend/Engine.php

```diff
@@ -117,49 +128,49 @@ class Engine
     /**
      * The used code node builder.
      *
-     * @var PHPBuilder<\PDepend\Source\AST\ASTNamespace>|null
+     * @var null|PHPBuilder<ASTNamespace>
```

Comment:
```suggestion
     * @var PHPBuilder<ASTNamespace>|null
```
I would keep the order with null always at the end.

Reply:
Ok, I can do that as a fullow up once this is merged.